project/AppDependencies.scala

```diff
@@ -4,7 +4,7 @@ import sbt._
 object AppDependencies {
 
   val bootstrapVersion = "5.16.0"
```

Comment:
OBSERVATION: could you up the other dependencies too?

Reply:
All done